_developers/developer-guide.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: "Developer Guide"
+title: "Projects"
 lang: en
 ref: developer-guide
 permalink: /developers/
-excerpt: "Developer Guide"
+excerpt: "Projects"
```

Comment:
We have bunch of developer related stuff,  in this page, I don't think change it to projects is good enough.

Reply:
@WillemJiang Thanks. The "Developer Guide" label was renamed "Community". Here created a label called "Projects" to make it easier for users to find our projects.